README.rst

```diff
@@ -2,6 +2,8 @@
 pynwb
 ===============================
 
+Documentation of the PyNWB can be found at http://pynwb.readthedocs.io/.
```

Comment:
"of the PyNWB"  should be either "of PyNWB" or "of the PyNWB API"

Reply:
`of PyNWB` 👍 

The documentation includes both API and more general documentation